src/main/java/loanbook/commons/core/Messages.java

```diff
@@ -12,4 +12,12 @@
     public static final String MESSAGE_INVALID_PASSWORD = "The password provided is incorrect!";
     public static final String MESSAGE_INVALID_OLD_PASS = "The old password entered is incorrect!";
     public static final String MESSAGE_SAME_AS_CURRENT_PASSWORD = "The password provided is same as before!";
+    public static final String MESSAGE_AUTHEN_FAILURE = "Connection to your email failed! The password is wrong "
```

Comment:
I am pretty sure networks is going to be an issue too. Can we have an error message to account for that?

Reply:
Sure, thank you :)